includes/tracks/class-wc-site-tracking.php

```diff
@@ -18,17 +18,21 @@ class WC_Site_Tracking {
 	 */
 	public static function is_tracking_enabled() {
 		/**
-		 * Don't track users who haven't opted-in to tracking or if a filter
-		 * has been applied to turn it off.
+		 * Don't track users if a filter has been applied to turn it off.
+		 * `woocommerce_apply_tracking` will be deprecated. Please use
+		 * `woocommerce_apply_user_tracking` instead.
 		 */
-
-		if ( ! apply_filters( 'woocommerce_apply_user_tracking', true ) ) {
+		if ( ! apply_filters( 'woocommerce_apply_user_tracking', true ) && ! apply_filters( 'woocommerce_apply_tracking', true ) ) {
```

Comment:
Now both filters need to deactivated in order to disable it.
Need to replace `&&` to `||`.

Reply:
Oh right, whoops. Thank you